### ISSUE TITLE
Cannot run ionic cordova prepare inside Docker container

### ISSUE THREAD

**Issue by rolandjitsu** [ionic-team/ionic-cli#2788]
**Description:**

I'm getting an error while running the `ionic cordova prepare` command inside a docker container.

**Steps to Reproduce:**

Example repo: 

Dockerfile:
```dockerfile
FROM ubuntu:trusty
RUN rm /bin/sh && ln -s /bin/bash /bin/sh

ARG npm_user
ENV NPM_USER=$npm_user
ARG npm_token
ENV NPM_TOKEN=$npm_token


# Java 8
# Source at https://github.com/webratio/docker/blob/master/java/7/Dockerfile,
# official image at https://github.com/dockerfile/java/blob/master/oracle-java8/Dockerfile.
RUN \
    apt-get -y update && \
    apt-get -y install software-properties-common && \
    add-apt-repository -y ppa:webupd8team/java && \
    apt-get -y update && \
    echo oracle-java8-installer shared/accepted-oracle-license-v1-1 select true | /usr/bin/debconf-set-selections && \
    apt-get -y install oracle-java8-installer && \
    apt-get -y remove software-properties-common && \
    apt-get -y autoremove && \
    apt-get clean
ENV JAVA_HOME /usr/lib/jvm/java-8-oracle

# Check JDK installation
RUN echo "JDK $(which java)"


# Android SDK
# https://developer.android.com/studio/index.html
# Source at https://github.com/webratio/docker/blob/master/android-sdk/r24.4.1/Dockerfile
ENV ANDROID_SDK_VERSION r24.4.1
ENV ANDROID_BUILD_TOOLS_VERSION 23.0.2

# Install i386 architecture required for running 32 bit Android tools
RUN dpkg --add-architecture i386 && \
    apt-get -y update && \
    apt-get -y install libc6:i386 libncurses5:i386 libstdc++6:i386 lib32z1 && \
    rm -rf /var/lib/apt/lists/* && \
    apt-get -y autoremove && \
    apt-get clean

# Install Android SDK
# For API levels, see https://developer.android.com/guide/topics/manifest/uses-sdk-element.html
ENV ANDROID_SDK_FILENAME android-sdk_${ANDROID_SDK_VERSION}-linux.tgz
ENV ANDROID_SDK_URL http://dl.google.com/android/${ANDROID_SDK_FILENAME}
ENV ANDROID_API_LEVELS android-19,android-20,android-21,android-22,android-23,android-24,android-25,android-26
ENV ANDROID_HOME /opt/android-sdk-linux
ENV PATH ${PATH}:${ANDROID_HOME}/tools:${ANDROID_HOME}/platform-tools
RUN cd /opt && \
    wget -q ${ANDROID_SDK_URL} && \
    tar -xzf ${ANDROID_SDK_FILENAME} && \
    rm ${ANDROID_SDK_FILENAME} && \
    echo y | android update sdk --no-ui -a --filter tools,platform-tools,${ANDROID_API_LEVELS},build-tools-${ANDROID_BUILD_TOOLS_VERSION},extra-android-m2repository,extra-android-support

# Check SDK installation
RUN echo "ADB $(which adb)"


# NVM/Node
# Install Node version manager
# https://github.com/creationix/nvm#install-script
# https://github.com/creationix/nvm#nvmrc
# TODO: Read node version from .nvmrc
ENV NVM_DIR /usr/local/nvm
ENV NODE_VERSION 7.9.0

RUN wget -qO- https://raw.githubusercontent.com/creationix/nvm/v0.33.5/install.sh | bash && \
    source $NVM_DIR/nvm.sh && \
    nvm install $NODE_VERSION && \
    nvm alias default $NODE_VERSION && \
    nvm use default && \
    # Install latest NPM
    npm i -g npm@latest

# Add Node binary to PATH
# ENV NODE_PATH $NVM_DIR/v$NODE_VERSION/lib/node_modules
ENV PATH $NVM_DIR/versions/node/v$NODE_VERSION/bin:$PATH

# Check Node installation
RUN echo "Node $(which node)"


# Yarn
# https://yarnpkg.com/en/docs/install#alternatives-tab
RUN apt-get -y update && \
    apt-get -y install apt-transport-https curl && \
    curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - && \
    echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list && \
    apt-get -y update && \
    apt-get -y install yarn

# Check Yarn installation
RUN echo "Yarn $(which yarn)"


# Add source and set working dir
ADD . /app
WORKDIR /app


# Deps
# IMPORTANT: Please make sure that $NPM_USER and $NPM_TOKEN are setup in your env
# Provide the NPM credentials via `docker build . --build-arg npm_user=<some user> --build-arg npm_token=<some token>`
# Check NPM config
RUN echo "NPM Email $(npm config get email)" && \
    echo "NPM Auth $(npm config get always-auth)"

# Install deps, platforms and plugins
RUN npm cache verify && \
    npm install && \
    ./node_modules/.bin/ionic cordova platform add android
    # ./node_modules/.bin/ionic cordova prepare


# VOLUME ["/app"]
# WORKDIR app

# EXPOSE 8100 35729 5037 9222 5554 5555

# CMD ["ionic", "serve"]
```

1. Install Docker
2. `docker build . -t ionic --build-arg npm_user=$NPM_USER --build-arg npm_token=$NPM_TOKEN`

**Output:**

```
rror: ENOENT: no such file or directory, open '/root/.ionic/daemon.log'
The command '/bin/sh -c npm cache verify &&     npm install &&     ./node_modules/.bin/ionic cordova platform add android' returned a non-zero code: 1
```

**My `ionic info`:**

**Other Information:**

**Comment by dwieeb** [comment]
Thanks @rolandjitsu! This should now be fixed in 3.13.0.

**Comment by rolandjitsu** [comment]
@dwieeb will check it out soon, thanks!
